python/cudf/cudf/core/reshape.py

```diff
@@ -196,16 +191,16 @@ def concat(objs, axis=0, ignore_index=False, sort=None):
     for o in objs:
         if isinstance(o, cudf.MultiIndex):
             typs.add(cudf.MultiIndex)
-        if issubclass(type(o), Index):
+        if issubclass(type(o), cudf.Index):
             typs.add(type(o))
-        elif isinstance(o, DataFrame):
-            typs.add(DataFrame)
-        elif isinstance(o, Series):
-            typs.add(Series)
+        elif isinstance(o, cudf.DataFrame):
+            typs.add(cudf.DataFrame)
+        elif isinstance(o, cudf.Series):
+            typs.add(cudf.Series)
```

Comment:
I realize this isn't your change, but... is `typs` a typo?

Reply:
I think the intent is to avoid confusion with the [`types`](https://docs.python.org/3.8/library/types.html) module